spec/v1.0-draft/digest_set.md

```diff
@@ -18,23 +18,22 @@ metadata object.
 ## Fields
 
 A DigestSet is represented as a _JSON object_ mapping algorithm name to
-lowercase hex-encoded value. Usually there is just a single key/value pair,
-but multiple entries MAY be used for algorithm agility.
-
-> **TODO**: Add language for supporting base64-encoded digest values (re: goModuleH1).
+a string value. Usually there is just a single key/value pair, but
+multiple entries MAY be used for algorithm agility.
 
 Supported algorithms:
 
--   Standard cryptographic hash algorithms, for cases when the method
-    of serialization is obvious or well known:
+-   Standard cryptographic hash algorithms with lowercase hex-encoded
+    values, for cases when the method of serialization is obvious or
+    well known:
     `sha256`, `sha224`, `sha384`, `sha512`, `sha512_224`, `sha512_256`,
     `sha3_224`, `sha3_256`, `sha3_384`, `sha3_512`, `shake128`, `shake256`,
     `blake2b`, `blake2s`, `ripemd160`, `sm3`, `gost`, `sha1`, `md5`
 
 -   `goModuleH1`: The go module [directory Hash1][], omitting the "h1:"
-    prefix and output in hexadecimal instead of base64. Can be computed
-    over a directory named `name@version`, or the contents of zip file
-    containing such a directory:
+    prefix and output in lowercase hexadecimal instead of base64. Can
```

Comment:
Since the `Hash1` function outputs a base64-encoded string, should we allow base64 encoding for this algorithm?

Reply:
That would make sense, I wonder if it would have any backwards compatibility issues?  E.g. we'd allow EITHER lowercase-hex or base64 and users would need to support both.

@MarkLodato WDYT?

Reply:
That seems like it makes the producer's job ever-so-slightly easier but the consumer's job much harder, especially if most of the time they receive one encoding but then they suddenly get a different one. I'm inclined to pick one.

Between base64 and hex, I don't feel too strongly.

Reply:
I think the best approach might be

1. Leave goModuleH1 as lowercase hex so that existing use cases can stay the same and we can unblock 1.0 with minimal risk.
2. In the future, if needed, add goModuleH1+base64 (per @marcelamelara's suggestion) so that people can specify _both_ goModuleH1 (lowercase hex) and goModuleH1+base64.

Reply:
Might be good to add a comment somewhere in the docs, or open an issue for this, just so we don't forget.

Reply:
Agree with Mark: generally, for security reasons (e.g., for the same reasons DSSE was formulated), best to decode using one expected format

Reply:
Ok, I'll leave it as is, without a note until we actually need something else.

Thanks everyone!